src/mixins/nft.js

```diff
@@ -262,12 +278,20 @@ export default {
         await this.initIfNecessary();
         const balance = await getAccountBalance(this.getAddress);
         if (balance === '0') {
+          logTrackerEvent(
+            this,
+            'NFT',
+            'NFTTransfer-insufficient',
+            this.getAddress,
+            1
+          );
           this.uiSetTxError('INSUFFICIENT_BALANCE');
           this.uiSetTxStatus(TX_STATUS.INSUFFICIENT);
           return;
         }
 
         this.uiSetTxStatus(TX_STATUS.SIGN);
+        logTrackerEvent(this, 'NFT', 'NFTTransfer-sendGrant', this.classId, 1);
```

Comment:
Maybe put the event to the page instead of this mixin, so that we can know where the event triggered from

Reply:
We have logged the event everywhere before `collectNFT()` is triggered, so I think we can know where this `sendGrand` event came from

[campaign page](https://github.com/likecoin/liker-land/blob/master/src/components/NFTCampaign/Item.vue#:~:text=handleClickCollect()%20%7B,%7D%2C)
[portfolio page](https://github.com/likecoin/liker-land/blob/master/src/components/NFTCampaign/Item.vue#:~:text=handleClickCollect()%20%7B,%7D%2C)
[details page](https://github.com/likecoin/liker-land/blob/master/src/pages/nft/class/_classId.vue#:~:text=%7D%2C-,handleCollectFromPreviewSection()%20%7B,%7D%2C,-async%20getLIKEPrice()%20%7B)